src/CommonITILObject.php

```diff
@@ -497,6 +495,8 @@ protected function setPredefinedFields(ITILTemplate $tt, array &$options, array
             $options['entities_id']         = $problem->fields['entities_id'];
         }
 
+        $this->initForm($this->fields['id'], $options);
```

Comment:
Calling `initForm` in `setPredefinedFields` seems weird to me.

Reply:
oh i missed that. Indeed, initForm should be kept in showForm function